GraphicEditor/GraphicEditor.csproj

```diff
@@ -14,6 +14,8 @@
     <WarningLevel>4</WarningLevel>
     <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <Deterministic>true</Deterministic>
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
```

Comment:
Unused?

Reply:
This change occurred automatically. It probably appeared after importing MaterialDesign library via NuGet Packet Manager.

Reply:
It's fine then